src/main/resources/db/changelog/changeset/0046_remove_feeback_related_columns/removeIs_peer_chatFromSession-rollback.sql

```diff
@@ -0,0 +1,6 @@
+ALTER TABLE `userservice`.`session`
```

Comment:
Do we not also remove the rc_feedback_group_id -> so a rollback is not possible with your where-clause, isn't it?
With this remove process, I would actually tend to think that there is no rollback, as it is not possible to restore the old state so the old data.
Therefore, my suggestion would be not to offer rollbacks here.

Reply:
I agree, we cannot offer rollback for the removed fields since the data will not exist there anymore.
I think we need to do a backup before running the migration, so, in the case of a rollback, we would still have the data. :)

Reply:
on the new caritas system there will be no data btw, so we need nothing here.